scripts/scripts.js

```diff
@@ -156,6 +156,29 @@ export async function fetchLanguagePlaceholders() {
   return {}; // default to empty object
 }
 
+/**
+ * Helper function to create DOM elements
+ * @param {string} tag DOM element to be created
+ * @param {Object} attributes attributes to be added
+ * @param {HTMLElement|SVGElement|string} html HTML or SVG to append to/after new element
+ */
+export function createTag(tag, attributes, html = undefined) {
+  const el = document.createElement(tag);
```

Comment:
We have dom-helper.js to created tags. Dont create new method.

Reply:
ok..didn't know that. Will do